lib/huginn_acumen_product_agent/concerns/acumen_product_query_concern.rb

```diff
@@ -125,6 +126,13 @@ def parse_product_request(products)
               })
             end
 
+            if field_value(p, 'Inv_Product.BO_Reason')
+              variant['offers'].push({
+                  '@type' => 'Offer',
+                  'availability' => field_value(p, 'Inv_Product.BO_Reason')
+              })
+            end
```

Comment:
@Jhalter5Stones I don't think you want to _create_ an Offer here.

The acumen agent _should_ be creating `Offer` instances somewhere else with the price fields. We should be _adding_ `availability` to those offers.